source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/ActionElementRendererFluentIconImageLoaderAsync.kt

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package io.adaptivecards.renderer
+
+import android.content.Context
+import android.graphics.Bitmap
+import android.graphics.Canvas
+import android.graphics.Matrix
+import android.graphics.drawable.BitmapDrawable
+import android.graphics.drawable.Drawable
+import android.view.View
+import android.widget.Button
+import com.caverock.androidsvg.SVG
+import io.adaptivecards.objectmodel.IconPlacement
+
+/**
+ * Responsible for rendering the Fluent Icon in the Action Element
+ **/
+class ActionElementRendererFluentIconImageLoaderAsync(
+    renderedCard: RenderedAdaptiveCard,
+    iconSize: Long,
+    view: View,
+    iconColor: String,
+    val iconPlacement: IconPlacement,
+    val padding: Long,
+    private val iconSizeFromConfig: Long
+): FluentIconImageLoaderAsync(renderedCard, iconSize, iconColor, view) {
+    override fun renderFluentIcon(drawable: Drawable?, flipInRtl: Boolean) {
+        val view = viewReference.get()
+        if (view != null && view is Button) {
+            val drawables = view.compoundDrawablesRelative
+
+            drawable?.let {
+
+                val flippedDrawable = flipDrawableHorizontally(it, view, flipInRtl)
+
+                if (iconPlacement == IconPlacement.AboveTitle) {
+                    drawables[1] = flippedDrawable
+                } else {
+                    drawables[0] = flippedDrawable
+                    view.compoundDrawablePadding = Util.dpToPixels(view.context, padding.toFloat())
+                }
+
+                view.setCompoundDrawablesRelativeWithIntrinsicBounds(
+                    drawables[0],
+                    drawables[1],
+                    drawables[2],
```

Comment:
it should be wrapped in drawable size check as well to avoid OutOfBounds exception

Reply:
drawable size check is not required. 

view.compoundDrawablesRelative returns drawables for the start, top, end, and bottom borders.